x11-misc/dmenu/dmenu-5.1.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8 ?

Reply:
savedconfig.eclass doesn't seem to support EAPIs higher than 7